### PR TITLE
Abuse ipdb context fix plus blacklist script

### PR DESCRIPTION
**Pull request by orenzohar**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/14742
fixes: https://github.com/demisto/etc/issues/14799

## Description
fixed context issues. added AbuseIPDB-PopulateIndicators script.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details: changed context outputs (approved).
    

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-AbuseDB.yml [hunk 1]
```diff
@@ -107,7 +108,7 @@ script:
         21 : 'Web_App_Attack',
         22 : 'SSH',
         23 : 'IoT_Targeted'
-    }
+    }script-AbuseIPDB-PopulateIndicators.yml
```
Comment (glicht) [inline Integrations/integration-AbuseDB.yml]: Not sure why you have here this yml file. Possibly a typo?

File: Integrations/integration-AbuseDB.yml [hunk 2]
```diff
@@ -161,26 +162,31 @@ script:
         if not isinstance(info, list):
             info = [info]
 
-        context, table, dbot_scores = [], [], []
+        context_ip_generic, context_ip, human_readable, dbot_scores = [], [], [], []
         for analysis in info:
-            analysis_info = {
+            ip_ec = {
+                "Address" : analysis.get("ipAddress"),
+                "Geo" : { "Country" : analysis.get("countryName") or analysis.get("countryCode") }
```
Comment (yuvalbenshalom) [inline Integrations/integration-AbuseDB.yml]: what happens if analysis don't have `countryName` or `countryCode`? we get `None`?

File: Scripts/script-AbuseIPDBPopulateIndicators.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+commonfields:
```
Comment (yuvalbenshalom) [inline Scripts/script-AbuseIPDBPopulateIndicators.yml]: please add a test

File: Integrations/integration-AbuseDB.yml (hunk 1 above)
Comment (orenzohar) [reply Integrations/integration-AbuseDB.yml]: yeah, thanks.

File: Integrations/integration-AbuseDB.yml (hunk 2 above)
Comment (orenzohar) [reply Integrations/integration-AbuseDB.yml]: createContext(categories, *removeNull=True*) should ignore it, but i will check.

Comment (orenzohar) [reply Integrations/integration-AbuseDB.yml]: ok it's showing as "null". I'll talk to you about it tomorrow.

File: Scripts/script-AbuseIPDBPopulateIndicators.yml (hunk 3 above)
Comment (orenzohar) [reply Scripts/script-AbuseIPDBPopulateIndicators.yml]: done.

**Pull request merged**